setup.py

```diff
@@ -13,6 +13,8 @@
 
 from coalib import VERSION, assert_supported_version, get_version
 from coalib.misc.BuildManPage import BuildManPage
+from coalib import get_current_commit, COMMIT_FILE, COALIB_DIR
```

Comment:
these imports should be near the other imports from the same module.
one way to do that is to move this line up one line.